### PR TITLE
New

### PR DESCRIPTION
**Pull request by andypasso**

### CODE REVIEW

**Comment by samgaco** [comment]
Currently reviewing this PR. Will update soon.

File: index.html [hunk 1]
```diff
@@ -0,0 +1,173 @@
...
+
+        <section class="hero your-verse">
+                <nav id="globalheader" class="apple globalheader-js svg globalheader-loaded">
+                        <!--googleoff: all-->
+                        <ul id="globalnav" class="navigation">
+                            <li id="gn-apple"><a href="/web/20140301004610/http://www.apple.com/"
+                                    class=""><span>Apple</span></a></li>
+                            <li id="gn-store"><a href="https://web.archive.org/web/20140301004610/http://store.apple.com/"><span>Store</span></a>
+                            </li>
+                            <li id="gn-mac"><a href="/web/20140301004610/http://www.apple.com/mac/"><span>Mac</span></a>
```
Comment (samgaco) [inline index.html]: You can just put a pound '#' on the 'href' category instead of that URL.

**Review by samgaco — CHANGES_REQUESTED** [review]
Hi @andynarf :hand: 

**Good work so far,** it feels like the original and checking your code I can tell there has been a lot of effort put into it, still, **there are some points to address;**

* You should use the focus effect on the` <input>` tag of the header.[ More info.](https://www.w3schools.com/cssref/sel_focus.asp)
* The 4 containers below (Iphone5, Happy Birthday, Mac., Supplier Responsibility) should be within the footer, and the images should be set as backgrounds of their respective div containers.
(Now they are enclosed in list tags but they should be div containers enclosed in` <a> `tags)
* Visualizing the index in the browser you can see that the horizontal scrolling bar makes an appearance, that shouldn't happen. It seems like the containers in "promos" are causing it.
* The main content of the site should be wrapped with `<main>` tags and have set _your_verse_hero.jpg_  as background.
* The elements in the navbar should be `<button>` tags. You can achieve the gradient effect with [linear gradient]( https://www.w3schools.com/cssref/func_linear-gradient.asp).

Working on this will surely help you grow more as a developer. Please :pray:  address these points and submit again for a code review. You are on the right path :muscle:

File: index.html (hunk 1 above)
Comment (andypasso) [reply index.html]: The elements in the navbar should be <button> tags. You can achieve the gradient effect with linear gradient. >>>
about this : I don't understand why the links should be buttons, I made it with anchors linking to the original section of the apple page  each. I did use linear-gradient on the background tho.

and about the links, i'd rather have working links to the original apple page.

thanks in advanced. =D

**Review by bolah2009 — APPROVED** [review]
# Congratulations @andynarf ! :tada:

Nice Job :muscle:

Good luck with your next project
Keep it up :star2:

Cheers
Bola

**Pull request merged**